src/feedback/index.ts

```diff
@@ -0,0 +1,12 @@
+import BaseApi from '../baseApi';
+import * as responseHandlers from '../utils/response/responseHandlers';
+import { FeedbackCreate, FeedbackResponse } from './types';
+
+export class FeedbackEndpoint extends BaseApi {
+  public readonly resourceName = 'feedbacks';
```

Comment:
Didn't we change that to `feedback`? 
```suggestion
  public readonly resourceName = 'feedback';
```

Reply:
Huh, indeed, I though we did.